net/message/blockHdr.go

```diff
@@ -229,6 +229,10 @@ func GetHeadersFromHash(startHash common.Uint256, stopHash common.Uint256) ([]le
 				return nil, 0, err
```

Comment:
Above function will return error if the header doesn't existed, so the patch looks not necessary, any comments?